compiler/modules/front/codeGen.bal

```diff
@@ -1688,7 +1696,7 @@ function getLangLibFunctionRef(CodeGenContext cx, bir:Operand target, string met
         string moduleName = t[0];
         bir:FunctionSignature? erasedSignature = getLangLibFunction(moduleName, methodName);
         if erasedSignature == () {
-            return cx.unimplementedErr(`unrecognized lang library function ${moduleName + ":" + methodName}`, pos);
+            return cx.unimplementedErr(`unrecognized lang library function ${moduleName + ":" + methodName}`, cx.qNameRange(pos));
```

Comment:
Comes from method call currently, so not a qname. But will in future, so pass in a Range.

Reply:
fixed